packages/documentation/src/stories/internet-header/components/header/header.stories.ts

```diff
@@ -0,0 +1,209 @@
+import { Meta, StoryObj, Args } from '@storybook/web-components';
+import { BADGE } from '../../../../../.storybook/constants';
+import { html } from 'lit';
+import { spread } from '@open-wc/lit-helpers';
+import { getAttributes } from '../../../../utils';
+import customConfig from './config/custom-config';
+import osFlyoutOverrides from './config/os-flyout-overrides';
+import languageSwitchOverrides from './config/language-switch-overrides';
+
+const meta: Meta = {
+  title: 'Internet Header/Header',
+  component: 'swisspost-internet-header',
+  parameters: {
+    badges: [BADGE.STABLE],
+    layout: 'fullscreen',
+    controls: { sort: 'alpha' },
+  },
+  render,
+  args: {
+    project: 'test',
+    environment: 'int01',
+    language: 'en',
+  },
+  argTypes: {
+    project: {
+      control: false,
+      table: {
+        category: 'Required props',
+      },
```

Comment:
Add an [SBType](https://storybook.js.org/docs/web-components/api/arg-types#type) to define the prop as required:
```js
project: {
  ...
  type: {
    name: 'string',
    required: true
  }
}
```